src/app/todo/page.tsx

```diff
@@ -0,0 +1,64 @@
+"use client";
+
+import { useRedirect } from "@/hooks/useRedirect";
+import { useUser } from "@/hooks/useUser";
+import { useState } from "react";
+import { v4 as uuid } from "uuid";
+
+export default function Todo() {
+  const [newTodo, setNewTodo] = useState("");
+  const [todos, setTodos] = useState<any[]>([]);
```

Comment:
Avoid using any as type. Please be more specific. Suggestion: Create a new type called ToDo as follows `type ToDo = {name: string; key:string completed: boolean}`